datarobot_provider/example_dags/model_compute_predictions_dag.py

```diff
@@ -29,10 +40,47 @@ def add_external_dataset(project_id=None, dataset_id=None):
         dataset_id=dataset_id,
     )
 
-    add_external_dataset_op
+    request_model_predictions_op = RequestModelPredictionsOperator(
+        task_id="request_model_predictions",
+        project_id=project_id,
+        model_id=model_id,
+        external_dataset_id=add_external_dataset_op.output,
+    )
+
+    model_predictions_sensor = DataRobotJobSensor(
+        task_id="model_predictions_complete",
+        project_id=project_id,
+        job_id=request_model_predictions_op.output,
+        poke_interval=5,
+        timeout=3600,
+    )
+
+    @task(task_id="example_custom_python_code")
+    def using_custom_python_code(project_id, predict_job_id, datarobot_conn_id='datarobot_default'):
+        """Example of using custom python code:"""
+
+        # Initialize DataRobot client
+        DataRobotHook(datarobot_conn_id).run()
+
+        # Should be used only with a very small datasets (not higher than a few megabytes).
```

Comment:
Maybe it would be good to move this comment up or duplicate it where the user specified their external dataset?

Reply:
I will add this in operators description in readme file